card-mod-graphql/lib/graph_q_l/types/query.rb

```diff
@@ -22,8 +22,8 @@ def card **mark
         ok_card nil, **mark
       end
 
-      def cards name: nil, type: nil, limit: 10, offset: 0
-        card_search name, type, limit, offset
+      def cards codename: nil, filter: nil, limit: 10, offset: 0
+        card_search codename, filter, limit, offset
```

Comment:
It looks like this still calls card_search but card_search isn't hear any more, right?

Can you add a test for this?

Reply:
Also, if `codename` refers to a cardtype, can we call it `type_code` instead?

Reply:
I removed card_search method